src/effects/Equalization.h

```diff
@@ -274,6 +276,9 @@ class EffectEqualization : public Effect,
    wxBoxSizer *szrM;
 #endif
 
+   CapturedParameters<EffectEqualization> mParameters;
```

Comment:
What is this member for? we already have the static capture `parameters` in `EffectEqualization::Parameters`

Reply:
Good find.  That was left over from an earlier version of this pull request, in which the parameters object still needed to capture `this` and could not be a static.  I left the same in TruncSilence.h.